src/user-agent.ts

```diff
@@ -4,4 +4,15 @@
  * user agent of sdk
  * @hidden
  */
-export const USER_AGENT = "sdk-js-2.0.0-alpha.18";
+import { isSSR } from "./index";
+
+const scriptTag = (document.currentScript as HTMLScriptElement | null)?.src;
+let suffix = "";
+const ssr = isSSR();
+if (scriptTag) {
+  const source = new URL(scriptTag).host;
```

Comment:
Unfortunately I don't think we can use `new URL` because of IE [1]. And I believe we need to support IE for this package correct @benhinchley ?

[1] https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
My stance on this would be that we note which polyfills would be required if end users want to support vintage browsers like IE11 or we end up bloating the code for the other 99% of consumers. It's only going to drop in numbers and we wouldn't want to use a hacky method to do the same (create an `<a>` set the URL and read the `location`) I'd say or it'll just hang around forever.

Reply:
Yep I guess we could just bump the minor version with this change and note that IE is not supported or that IE users need to include the polyfill. That works for me.

Reply:
Yeah, we need to get `2.x` out anyway. That's part of the reason we're adding the source in, so we can find folks using it via unpkg etc and unversioned 👍